### PR TITLE
PlayStatusMessage

### PR DESCRIPTION
**Pull request by dgreenhalgh**
Keeps everyone's play/pause status in sync

### CODE REVIEW

File: SoundStream/src/com/lastcrusade/soundstream/audio/RemoteAudioPlayer.java [hunk 1]
```diff
@@ -39,4 +47,21 @@ public void pause() {
     public void skip() {
         this.application.getMessagingService().sendSkipMessage();
     }
+    
+    private void registerReceivers() {
+    	this.registrar = new BroadcastRegistrar();
+    	this.registrar.addAction(MessagingService.ACTION_PLAY_STATUS_MESSAGE, new IBroadcastActionHandler() {
```
Comment (reidbaker) [inline SoundStream/src/com/lastcrusade/soundstream/audio/RemoteAudioPlayer.java]: tad long

File: SoundStream/src/com/lastcrusade/soundstream/net/message/PlayStatusMessage.java [hunk 2]
```diff
@@ -0,0 +1,51 @@
...
+public class PlayStatusMessage extends ADataMessage {
+	
+	private final String TAG = PlayStatusMessage.class.getName();
+	private final String PLAY_MESSAGE = "Play";
+	private final String PAUSE_MESSAGE = "Pause";
+	private String string = "";
+	
+	public PlayStatusMessage() {}
+
+	public PlayStatusMessage(String playStatusMessage) {
```
Comment (theJenix) [inline SoundStream/src/com/lastcrusade/soundstream/net/message/PlayStatusMessage.java:24]: Please replace with boolean to represent playing or not playing.

File: SoundStream/src/com/lastcrusade/soundstream/service/PlaylistService.java [hunk 3]
```diff
@@ -111,6 +111,9 @@ public void onReceiveAction(Context context, Intent intent) {
             @Override
             public void onReceiveAction(Context context, Intent intent) {
                 mPlaylist.moveNext();
+                // TODO: Right now, we don't move automatically to the next song.
```
Comment (reidbaker) [inline SoundStream/src/com/lastcrusade/soundstream/service/PlaylistService.java:114]: Great comment

File: SoundStream/src/com/lastcrusade/soundstream/net/message/PlayStatusMessage.java (hunk 2 above)
Comment (dgreenhalgh) [inline SoundStream/src/com/lastcrusade/soundstream/net/message/PlayStatusMessage.java:24]: Added TODO for potential refactor :horse: :shit:

**Comment by theJenix** [comment]
Looks good to me...I'm pulling it in.

**Pull request merged**